tools/moFF/moff.xml

```diff
@@ -210,30 +233,64 @@
                 <expand macro="raw_input_macro" allow_multiple="false"/>
                 <param argument="--tol" type="float" value="10" label="Tolerance parameter"
                     help="Specify the tolerance parameter in ppm." />
-                <param argument="--rt_w" type="float" value="3.0" label="Retention time window"
+                <param argument="--mbr" type="text" value="off" label="moFF workflow"
+                    help="select the moFF workflow" />
+                <param argument="--xic_length" type="float" value="3.0" label="retention time windows for XiC"
                     help="Specify rt window for xic in minutes." />
-                <param argument="--rt_p" type="float" value="1" label="Time window for the peak"
-                    help="Specify the time windows for the peak in minutes." />
+                <param argument="--rt_peak_win" type="float" value="1" label="retention time window for apex detection"
+					help="Specify rt window for the peak in minutes." />
+				<param argument="--rt_peak_win_match" type="float" value="1.2" label="retention time window for the matched peak"
+                    help="Specify the retention time window for the matched peak in minutes." />
+                <conditional name="match_filter">
```

Comment:
This conditional is used 3 times, we should move this into a macro for reuse. Also, it would be great to change this from a boolean to a select.

Reply:
In my last commit , I created a macro  for the conditional. This should be use only two times but I have found 3 occurences.

Reply:
there was a duplicated conditional in the apex case

Reply:
@bgruening why would it be better with a select? the options are only yes or no, right?

Reply:
Some weeks ago @bgruening recommended me in another tool the same because of this:

https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html?highlight=boolean#booleans

ie:

Options that are conditionally hidden (using the <conditional> element) should use a select param type and not a boolean. The user may not expect a boolean checkbox to change the content of a form.

Reply:
ok, thanks for the input, I can change it.